opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -508,6 +518,136 @@ message DoubleHistogramDataPoint {
   repeated DoubleExemplar exemplars = 8;
 }
 
+// SummaryDataPoint is a single data point in a timeseries that describes the
+// time-varying values of a Summary metric.
+message IntSummaryDataPoint {
+  // The set of labels that uniquely identify this timeseries.
+  repeated opentelemetry.proto.common.v1.StringKeyValue labels = 1;
+
+  // start_time_unix_nano is the last time when the aggregation value was reset
+  // to "zero". For some metric types this is ignored, see data types for more
+  // details.
+  //
+  // The aggregation value is over the time interval (start_time_unix_nano,
+  // time_unix_nano].
+  //
+  // Value is UNIX Epoch time in nanoseconds since 00:00:00 UTC on 1 January
+  // 1970.
+  //
+  // Value of 0 indicates that the timestamp is unspecified. In that case the
+  // timestamp may be decided by the backend.
+  fixed64 start_time_unix_nano = 2;
+
+  // time_unix_nano is the moment when this aggregation value was reported.
+  //
+  // Value is UNIX Epoch time in nanoseconds since 00:00:00 UTC on 1 January
+  // 1970.
+  fixed64 time_unix_nano = 3;
+
+  // count is the number of values in the population. Must be non-negative.
+  fixed64 count = 4;
+
+  // sum of the values in the population. If count is zero then this field
+  // must be zero.
+  sfixed64 sum = 5;
+
+  // min of the values in the population.
+  sfixed64 min = 6;
+
+  // max of the values in the population.
+  sfixed64 max = 7;
+
```

Comment:
I would not add this for the moment, we can followup. The purpose of this is to be compatible with OpenMetrics/Prometheus and they don't define that.

Reply:
I've removed them, thanks for the comment